.github/scripts/check-engines-cli.sh

```diff
@@ -84,7 +84,7 @@ elif [ $CLI_QUERY_ENGINE_TYPE == "library" ]; then
       qe_location2="node_modules\prisma\node_modules\engines\query_engine-windows.dll.node"
       ;;
   esac
-elif [ $CLI_QUERY_ENGINE_TYPE == "dataproxy" ]; then
+elif [ $CLI_QUERY_ENGINE_TYPE == "<dataproxy>" ]; then
```

Comment:
What is this about? Just disabling the code without removing it?

Reply:
Or reusing the old code to easily trigger that logic via the old env var instead of completely re-factoring ecosystem-tests?

Reply:
> Or reusing the old code to easily trigger that logic via the old env var instead of completely re-factoring ecosystem-tests?

Yes, we can improve that later.